core/src/main/java/org/opensearch/sql/expression/function/PPLFuncImpTable.java

```diff
@@ -270,11 +273,14 @@ void populate() {
       registerOperator(RIGHT, SqlLibraryOperators.RIGHT);
       registerOperator(LEFT, SqlLibraryOperators.LEFT);
       registerOperator(LOG2, SqlLibraryOperators.LOG2);
+      registerOperator(MD5, SqlLibraryOperators.MD5);
+      registerOperator(SHA1, SqlLibraryOperators.SHA1);
       registerOperator(INTERNAL_REGEXP_EXTRACT, SqlLibraryOperators.REGEXP_EXTRACT);
       registerOperator(INTERNAL_REGEXP_REPLACE_2, SqlLibraryOperators.REGEXP_REPLACE_2);
 
       // Register PPL UDF operator
       registerOperator(SPAN, PPLBuiltinOperators.SPAN);
+      registerOperator(SHA2, PPLBuiltinOperators.SHA2);
```

Comment:
Seems the `SHA2` is the only one we built by ourselves. So I checked why PPL Spark supported `SHA2`, I found there was no requirement for this. This [issue](https://github.com/opensearch-project/opensearch-spark/issues/759) mentioned a link from Splunk, but there was no `SHA2` in Splunk either. @penghuo how about remove the `SHA2` both in Spark and OpenSearch.

Reply:
I'm okay with using SHA2, as it provides more functionality.